src/modules/Dashboard/features/Respondents/Respondents.tsx

```diff
@@ -309,14 +331,13 @@ export const Respondents = () => {
         },
       }),
       status: {
-        content: () =>
-          status !== RespondentStatus.Invited && (
-            <StatusFlag
-              status={status}
-              onInviteClick={() => handleInviteClick({ respondentOrSubjectId, email })}
-              isInviteDisabled={!filteredRespondents?.[respondentOrSubjectId]?.editable.length}
-            />
```

Comment:
Is it perhaps worth removing the `StatusFlag` component entirely at this point? It looks like the only place it's rendered now is in its own test file. Seems like it might be simple enough to cleanup if we consider this an "unsupported" component now.

Reply:
Good catch, removing.